publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RMQHelper.java

```diff
@@ -86,7 +88,7 @@ private void protocolInit(String protocol) throws RemRemPublishException{
         rabbitMqPropertiesMap.get(protocol).init();
     }
 
-    public void send(String routingKey, String msg, MsgService msgService) throws IOException {
+    public void send(String routingKey, String msg, MsgService msgService) throws IOException, NackException, TimeoutException, RemRemPublishException {
         String protocol = msgService.getServiceName();
         if(rabbitMqPropertiesMap.get(protocol) != null) {
```

Comment:
Why get(protocol) is called over and over? A local variable holding result of get(protocol) would improve efficiency and readability of the code. There is a lot of similarly structured code in other files, too.

Reply:
removed multiple calls of get(protocol)